docs/tutorials/auth.md

```diff
@@ -311,12 +311,13 @@ secret as follows:
 skipper -oauth2-client-id-file=/path/to/client_id \
     -oauth2-client-secret-file=/path/to/client_secret \
     -oauth2-secret-file=/path/to/cookie_encryption_secret \
-    -credentials-paths=/path/to/ \
     -credentials-update-interval=30s
 ```
 
-You must set the `-credentials-paths` argument to the directory containing the secrets. You 
-can modify the secret update interval using the `-credentials-update-interval` argument. In
+Care must be taken when used in conjunction with `-credentials-paths` option because files
+from `-credentials-paths` are available to `bearerinjector` filter.
```

Comment:
Does this hold true?
Your current change has 2 non shared SecretsPaths instances that are not sharing the files.

Reply:
Yes, `SecretsPaths` are separate but they may scan the same folder - lets say one configures
1. ` -credentials-paths=/tmp/secrets`
2. `-oauth2-client-secret-file=/tmp/secrets/grant.secret`

I.e. grant secret is located in credentials path an therefore is available to `bearerinjector`.

Reply:
I have added demo example

Reply:
Ok yes thanks that makes sense if you use the same path. We should state that you should not use the same path for these.

Reply:
I have added
> That is `-credentials-paths=/path/to` in above example will expose grant files to `bearerinjector` filter.